src/main/java/ganymedes01/etfuturum/recipes/ModRecipes.java

```diff
@@ -998,6 +998,12 @@ ore_dyes[dye], new ItemStack(Blocks.sand, 1, 0), new ItemStack(Blocks.sand, 1, 0
 			}
 		}
 
+		if (Loader.isModLoaded("BigReactors")) {
```

Comment:
Needs to use [ModsList](https://github.com/Roadhog360/Et-Futurum-Requiem/blob/master/src/main/java/ganymedes01/etfuturum/compat/ModsList.java)